25_exceptions_lab/02.py

```diff
@@ -1,7 +1,14 @@
-"""
-Write a program that takes a file name
-and prints line count for the file
-
-Alert the user politely if there was any problem opening the file
-"""
-
+_author_ = 'mariag'
+
+import sys
+import os
+
+file = sys.argv[1]
+
+print file
+
+try:
+    lines = sum(1 for line in open(file))
+    print "Amount of lines in {} is {}".format(file, lines)
+except IOError as e:
+    print "Sorry, file {} not found".format(file)
```

Comment:
👍 though failure to open a file might be for other reasons as well, not just that it cannot be found.